mobile/src/core/store/app/reducer.ts

```diff
@@ -1,22 +1,35 @@
-import { AppSettings, Reducer } from '../../interfaces';
+import { BurstService } from '@burstjs/core';
+import { AppSettings, BurstSettings, Reducer } from '../../interfaces';
 import { createReducers } from '../../utils/store';
 import { actionTypes } from './actionTypes';
 
 export interface AppReduxState {
   isAppLoaded: boolean;
   appSettings: AppSettings;
+  burstService: BurstService;
 }
 
 export function getDefaultAppSettings (): AppSettings {
   return {
-    passcodeTime: 600000 // 10 min
+    passcodeTime: 600000, // 10 min,
+    burstSettings: getDefaultBurstSettings()
+  };
+}
+
+export function getDefaultBurstSettings (): BurstSettings {
+  return {
+    nodeHost: 'https://wallet1.burst-team.us:2083',
+    apiRootUrl: '/burst'
```

Comment:
On the angular app, we have these coming from an `environment.ts` file which is nice because we can use testnet for local dev and mainnet for production builds.

Reply:
I keep in mind settings screen with on-the-fly apply

Reply:
But .env is a thing, can agree

Reply:
btw dded .env